cdap-unit-test/src/test/java/co/cask/cdap/security/AuthorizationTest.java

```diff
@@ -358,20 +355,16 @@ public void testApps() throws Exception {
 
   @Test
   public void testArtifacts() throws Exception {
-    ArtifactId appArtifact = new ArtifactId("app-artifact", new ArtifactVersion("1.1.1"), ArtifactScope.USER);
     try {
-      NamespacedArtifactId defaultNsArtifact = new NamespacedArtifactId(
-        NamespaceId.DEFAULT.getNamespace(), appArtifact.getName(), appArtifact.getVersion().getVersion());
+      ArtifactId defaultNsArtifact = new ArtifactId(NamespaceId.DEFAULT.getNamespace(), "app-artifact", "1.1.1");
```

Comment:
nit: existing code, but this can be more concise if you do ``NamespaceId.DEFAULT.artifact("app-artifact", "1.1.1")``